.github/workflows/techdocs.yaml

```diff
@@ -55,6 +55,10 @@ jobs:
               - docker-compose/Dockerfile
             markdown:
               - added|modified: '**/*.md'
+              - added|modified: '**/*.jpg'
+              - added|modified: '**/*.JPG'
+              - added|modified: '**/*.png'
+              - added|modified: '**/*.svg'
```

Comment:
You want to move these to the build filter, the markdown filter is used to validate markdown files.